DEVELOPMENT.md

```diff
@@ -183,25 +227,22 @@ npm run typecheck
 
 You can test these packages locally by linking them, or by using a local npm registry such as Verdaccio.
 
-#### Using linking (recommended for local development)
+#### Package linking
 
 1. Follow steps 1–3 in [Setup](#setup).
 2. Run `npm link -w snapwp` to link the package into your global dependencies.
 3. Verify the CLI by running `snapwp`.
-4. Remove the global link with `npm r snapwp -g`.
+4. (Optional) Run `snapwp --proxy` to use proxied packages in the linked cli.
+5. Remove the global link with `npm r snapwp -g`.
 
-#### Using Verdaccio (recommended for pre-release testing)
+#### Verdaccio proxy registry
 
 1. Follow steps 1–4 in [Setup](#setup).
-2. Use `npm_config_registry=http://localhost:4873 npx snapwp` to run the local package.
+2. Use `npm_config_registry=http://localhost:4873 npx snapwp --proxy` to run the local package.
```

Comment:
In comparison with the #Setup steps: when do I need to set `npm_config_registry` and when do I need to use `--proxy`? (I'm assuming the `npx` is to use the unliked CLI with the proxied packages).

Reply:
Let me rephrase: (why) do we use `npm_config_registry` here and not in the setup steps?

Reply:
I will be using this section too. We were using `npm_config_registry` because of my personal bias against linking. Since this section was written, my stance is now "Linking to access binaries is fine but linking to get imports causes module resolution problems". 
I will be removing this section too.